packages/generators/package.json

```diff
@@ -26,6 +25,7 @@
     "babel-jest": "^23.6.0",
     "babel-plugin-inline-import": "^3.0.0",
     "documentation": "^8.0.0",
+    "ejs": "^2.6.1",
```

Comment:
Why did you do that? 
Ejs is used in src code.

Reply:
It replaced with string by babel. It doesn't use in dist. But trigger webpack warning in CW.